src/main/java/frc/robot/Controls.java

```diff
@@ -140,7 +142,10 @@ private void configureSuperStructureBindings() {
     // operator start button used for climb - bound in climb bindings
     operatorController
         .y()
-        .onTrue(Commands.runOnce(() -> branchHeight = BranchHeight.LEVEL_FOUR).withName("level 4"));
+        .onTrue(Commands.runOnce(() -> branchHeight = BranchHeight.LEVEL_FOUR).withName("level 4"))
+        .onTrue(
+            Commands.runOnce(() -> algaeIntakeHeight = AlgaeIntakeHeight.ALGAE_LEVEL_THREE_FOUR)
+                .withName("algae level 3-4"));
```

Comment:
Consider this to avoid creating and running a second command simply to set a variable, and updating the similar ones below for the `a`, `povUp`, and `povDown` buttons.
```suggestion
        .onTrue(Commands.runOnce(() -> {
            branchHeight = BranchHeight.LEVEL_FOUR;
            algaeIntakeHeight = AlgaeIntakeHeight.ALGAE_LEVEL_THREE_FOUR;
        }).withName("coral level4, algae level 3-4"));
```